cdm/src/main/java/thredds/inventory/NcmlCollectionReader.java

```diff
@@ -108,12 +90,12 @@ static public NcmlCollectionReader open(String ncmlLocation, Formatter errlog) t
       return null;
     }
 
-    Element scanElem = aggElem.getChild("scan", Catalog.ncmlNS);
-    if (scanElem == null) scanElem = aggElem.getChild("scanFmrc", Catalog.ncmlNS);
-    if (scanElem == null) {
-      errlog.format("NcML must have aggregation scan or scanFmrc element");
-      return null;
-    }
+    Element scanElem = aggElem.getChild("scan", myNS);
+    //if (scanElem == null) scanElem = aggElem.getChild("scanFmrc", Catalog.ncmlNS);
+    //if (scanElem == null) {
+    //  errlog.format("NcML must have aggregation scan or scanFmrc element");
+    //  return null;
+   //}

```

Comment:
Can this block be removed? Commented-out code is a form of technical debt that we have a lot of already.